lib/docker.rb

```diff
@@ -119,6 +119,29 @@ def ping(connection = self.connection)
     connection.get('/_ping')
   end
 
+  # Determine if the server is podman or docker.
+  def podman?(connection = self.connection)
+    @attrs ||= {}
```

Comment:
Sorry, I was thinking we could cache the `version` request coming from the connection. This is caching it on the top level module right now which can be shared across many connections.

Reply:
https://github.com/trevor-vaughan/docker-api/compare/podman-compat...swipely:podman-compat-cache-version Like this.